tests/components/elevenlabs/conftest.py

```diff
@@ -24,14 +24,18 @@ def mock_setup_entry() -> Generator[AsyncMock]:
         yield mock_setup_entry
 
 
-@pytest.fixture
-def mock_async_client() -> Generator[AsyncMock]:
-    """Override async ElevenLabs client."""
+def _client_mock():
     client_mock = AsyncMock()
     client_mock.voices.get_all.return_value = GetVoicesResponse(voices=MOCK_VOICES)
     client_mock.models.get_all.return_value = MOCK_MODELS
+    return client_mock
+
+
+@pytest.fixture
+def mock_async_client() -> Generator[AsyncMock]:
+    """Override async ElevenLabs client."""
     with patch(
-        "elevenlabs.AsyncElevenLabs", return_value=client_mock
+        "elevenlabs.AsyncElevenLabs", return_value=_client_mock()
```

Comment:
Let's also patch where we use it `homeassistant.component.elevenlabs.something.AsyncElevenLabs`

Reply:
Done, thanks!